client/jetpack-connect/site-url-input.jsx

```diff
@@ -122,7 +122,7 @@ class JetpackConnectSiteUrlInput extends PureComponent {
 						autoFocus="autofocus"
 						onChange={ onChange }
 						disabled={ isFetching }
-						placeholder={ translate( 'http://www.yoursite.com' ) }
+						placeholder={ translate( 'http://yourjetpack.blog' ) }
```

Comment:
Does this need to be translated?

Reply:
I would say not, since that would nullify the benefit of using a domain that points to something useful and relevant.

Reply:
Indeed, let's remove the translation.

Reply:
👍 for removing the `translate()` call.